kolibri/plugins/coach/assets/src/views/class-selector/index.vue

```diff
@@ -20,9 +21,8 @@
 
   import orderBy from 'lodash/orderBy';
   import { PageNames } from '../../constants';
-  import uiSelect from 'keen-ui/src/UiSelect';
+  import kSelect from 'kolibri.coreVue.components.kSelect';
```

Comment:
Is it possible to do `import { kSelect } from 'kolibri.coreVue.components`? Looking at apiSpec, it looks like  it can be done.

Reply:
I was just thinking about the other day. Something to address throughout the entire code base. Sorting imports would also be nice. https://eslint.org/docs/rules/sort-imports